components/eam/bld/build-namelist

```diff
@@ -737,6 +737,9 @@ if ($rad_volcaero){
     }
 }
 
+# Adding call for CLDERA prognostic stratospheric volcanic aerosol treatment
+my $cldera_strat_volc = $cfg->get('cldera_strat_volc'); #default should be no CLDERA treatment (0)
```

Comment:
So if I understand correctly, all that is required to run this is to append `-cldera_strat_volc` to `CAM_CONFIG_OPTS` at case setup?

Reply:
Yes. But you also have to set cldera_strat_volc = 1 in your runscript, otherwise the default is cldera_strat_volc = 0 and the model has the default renaming treatment.